db/schema.rb

```diff
@@ -124,6 +124,31 @@
     t.integer  "round",                    default: 1
   end
 
+  create_table "events", force: :cascade do |t|
```

Comment:
Uh oh, hm. I take it these were supposed to be removed?

You've got me here. Maybe I can remove the changes to the schema?